jupyterlab_latex/build.py

```diff
@@ -12,7 +12,7 @@
 from .util import run_command
 
 @contextmanager
-def latex_cleanup(cleanup=True, workdir='.', whitelist=None, greylist=None):
+def latex_cleanup(cleanup=False, workdir='.', whitelist=None, greylist=None):
```

Comment:
Does it work? It would seem this default is not used in:

https://github.com/jupyterlab/jupyterlab-latex/blob/db3dd9baf288db48983012f92c0a02ca9e4326cc/jupyterlab_latex/build.py#L306-L307


Can it be changed on the trait level?

https://github.com/jupyterlab/jupyterlab-latex/blob/58abc9c954d679c328a2e835b2961e606d65df2f/jupyterlab_latex/config.py#L29-L30

Reply:
> Does it work? It would seem this default is not used in:

Right! Sorry I didn't catch how it's working.
I have modified `config.py` and clarified (I hope) the documentation.
Thanks for your review!